src/property-filter/property-editor.tsx

```diff
@@ -43,6 +57,55 @@ export function PropertyEditorContent<TokenValue = any>({
   );
 }
 
+export function PropertyEditorContentEnum({
+  property,
+  filter,
+  value: unknownValue,
+  onChange,
+  asyncProps,
+  filteringOptions,
+  onLoadItems,
+}: {
+  property: InternalFilteringProperty;
+  filter: string;
+  value: null | string[];
+  onChange: (value: null | string[]) => void;
+  asyncProps: DropdownStatusProps;
+  filteringOptions: readonly InternalFilteringOption[];
+  onLoadItems?: NonCancelableEventHandler<LoadItemsDetail>;
+}) {
+  const valueOptions: readonly { value: string; label: string }[] = filteringOptions
+    .filter(option => option.property?.propertyKey === property.propertyKey)
+    .map(({ label, value }) => ({ label, value }));
+  const valueHandlers = useLoadItems(onLoadItems, '', property.externalProperty);
+  const value = !unknownValue ? [] : Array.isArray(unknownValue) ? unknownValue : [unknownValue];
+  const selectedOptions = valueOptions.filter(option => value.includes(option.value));
```

Comment:
Why not make value a set instead of an array so that we don't need to iterate it with `value.includes` for every element in `valueOptions`?

Reply:
The value is casted to an array if it is not, but it is expected to be an array. It is possible to introduce another var like `const valueSet = new Set(value)`, but I don't think it is much necessary in this case.